packages/pn-personafisica-webapp/src/component/Notifications/NotificationPayment.tsx

```diff
@@ -457,7 +457,7 @@ const NotificationPayment: React.FC<Props> = ({
     >
       <Paper sx={{ p: 3, mb: '1rem' }} className="paperContainer">
         <Grid container direction="row" justifyContent="space-between">
-          <Grid item xs={8} lg={8}>
+          <Grid item xs={8} lg={8} sx={{ marginBottom: '1rem' }}>
```

Comment:
Don't use rem or px or others, but use integer like 1 (= 8px), 2 (= 16px) and so on.
For this case, from figma, we have 16px and so we must use 2

`<Grid item xs={8} lg={8} mb={2}>`

Reply:
Changed.